recipes/ogre/1.x/test_package/conanfile.py

```diff
@@ -0,0 +1,19 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.definitions["OGRE_VERSION"] = tools.Version(self.deps_cpp_info["ogre"].version)
```

Comment:
There is a better trick. You can add the generator `cmake_find_package` then the `FindOGRE.cmake` generated by will contain `OGRE_VERSION`. But you don't need to update it, your version is good.